union-type.js

```diff
@@ -1,78 +1,130 @@
 var curryN = require('ramda/src/curryN');
 
-function isString(s) { return typeof s === 'string'; }
-function isNumber(n) { return typeof n === 'number'; }
-function isBoolean(b) { return typeof b === 'boolean'; }
-function isObject(value) {
-  var type = typeof value;
-  return !!value && (type == 'object' || type == 'function');
-}
-function isFunction(f) { return typeof f === 'function'; }
-var isArray = Array.isArray || function(a) { return 'length' in a; };
+if (process.env.NODE_ENV !== 'production') {
+  var isString = function(s) { return typeof s === 'string'; };
```

Comment:
is there any reason to not use `R.is` ?

```js
var is = require('ramda/src/is');

var isString = R.is(String);
var isNumber = R.is(Number);
...
```

either that or just use `R.is` directly.